storagenode/storageusage/storageusage.go

```diff
@@ -20,13 +20,15 @@ type DB interface {
 	// GetDailyTotal returns daily storage usage stamps summed across all known satellites
 	// for provided time range
 	GetDailyTotal(ctx context.Context, from, to time.Time) ([]Stamp, error)
+	// Summary returns aggregated storage usage across all satellites
+	Summary(ctx context.Context, from, to time.Time) (float64, error)
+	// SatelliteSummary returns aggregated storage usage for particular satellite
+	SatelliteSummary(ctx context.Context, satelliteID storj.NodeID, from, to time.Time) (float64, error)
 }
 
-// Stamp is storage usage stamp for satellite at some point in time
+// Stamp is storage usage stamp for satellite from interval start till next interval
```

Comment:
```suggestion
// Stamp is storage usage stamp for satellite from interval start till next interval.
```

Reply:
you could also say "until next interval"